Fix build failure by removing unneeded include
https://github.com/synfig/synfig/commit/dedb2f7f82a04c0cb3aad462bbe055412f3f5e18

The build failure is because extern "C" is being used to enclose a header
which later includes a C++ file (from icu).
--- src/synfig/savecanvas.cpp.orig
+++ src/synfig/savecanvas.cpp
@@ -66,9 +66,9 @@
 #include <ETL/stringf>
 #include "gradient.h"
 
-extern "C" {
+/*extern "C" {
 #include <libxml/tree.h>
-}
+}*/
 
 #endif
 
